app/controllers/index.rb

```diff
@@ -0,0 +1,101 @@
+# displays home page with login prompt
+get '/' do
+  erb :index
+end
+
+# analyzes and stores login/registration info
+post '/' do
+  # checking if registration info is correct
+  if params[:new_password].blank? && params[:new_username].blank?
+    @error = "Type in something you must!"
+    erb :index
+  elsif params[:new_password].blank?
+    @error = "PASSWORD IS NEEDED BRUH"
+    erb :index
+  else
+    @new_user = User.create(username: params[:new_username], password: params[:new_password])
+    @error = @new_user.errors[:username][0]
+    erb :index
+  end
+end
+
+get '/logout' do
+  session.delete :user_id
+  redirect '/'
+end
+
+# displays create_team form
+get '/create_team' do
+  erb :create_team
+end
+
+# allows user to enter team name
+post '/create_team' do
```

Comment:
this is called `create_team` but it looks like its a login action. What going on here?